src/frontend/apps/standalone_site/src/features/Playlist/features/UpdatePlaylist/components/UpdatePlaylistPage.tsx

```diff
@@ -94,6 +94,7 @@ export const UpdatePlaylistPage = ({ playlistId }: UpdatePlaylistPageProps) => {
               updatePlaylist({
                 organization: values.organizationId,
                 title: values.name,
+                retention_duration: values.retention_duration || null,
```

Comment:
Shouldn't you set the retention_duration inside the initialValues line 80 as well?